Lesson27/ezgi/update/script.js

```diff
@@ -0,0 +1,47 @@
+const urlParams = new URLSearchParams(window.location.search);//I couldn't figure out the reason to use this two lines
```

Comment:
When we redirect a user to the update post page, we pass a parameter in the URL. Check search bar in the browser when you get on the update post page and you'll see something like '?postId=2' in the end of the URL.

This way, we pass information about what post should be edited from main page to the update post page.

Reply:
I get it. thank you for the explanation :)